test/e2e/e2e_suite_test.go

```diff
@@ -203,3 +211,49 @@ func TestMain(m *testing.M) {
 	// Use Environment.Run to launch the test
 	os.Exit(testEnv.Run(m))
 }
+
+func useImageArchive(kindCluster *kind.Cluster, ctx context.Context) (context.Context, error) {
+	var err error
+	dname, err := os.MkdirTemp("", "etcdOperator")
+	if err != nil {
+		return ctx, nil
+	}
+
+	fname := filepath.Join(dname, archiveName)
+
+	defer func() {
+		err := os.RemoveAll(fname)
+		log.Printf("failed to remove %s: %s", fname, err)
+	}()
+
+	log.Println("Creating docker image archive...")
+	err = createImageArchive(fname)
+	if err != nil {
+		log.Printf("Failed to create archive %s:", err)
+		return ctx, err
+	}
+
+	log.Println("Loading docker image archive...")
+	ctx, err = loadImageArchive(*kindCluster, ctx, fname)
+	if err != nil {
+		return ctx, err
+	}
+	return ctx, nil
+}
+
+func createImageArchive(archivePath string) error {
+	// Save docker image archive
+	cmd := exec.Command("make", "docker-save", fmt.Sprintf("ARCHIVE_NAME=%s", archivePath))
```

Comment:
I think we should be passing the docker-image as defined in the test config.

```suggestion
	cmd := exec.Command("make", "docker-save", fmt.Sprintf("ARCHIVE_NAME=%s", archivePath), fmt.Sprintf("IMG=%s", dockerImage)
```

Reply:
I have an change planned for after this PR is merged that will make this redundant.

The change will set `dockerImage` to `os.Getenv("IMG")`. The idea being to reduce the need to modify `e2e_suite_test.go` to suit the local environment and then revert for before committing.

Reply:
Happy to address it as follow-up, no problem